feature/platform/tests/breakout_configuration/metadata.textproto

```diff
@@ -0,0 +1,16 @@
+# proto-file: github.com/openconfig/featureprofiles/proto/metadata.proto
+# proto-message: Metadata
+plan_id: "PLT-1.1"
+uuid: "6ac4eed0-9e04-46f5-85d8-ca88ee5fec7d"
+description: "Interface breakout Test"
+testbed: TESTBED_DUT_ATE_9LINKS
```

Comment:
What is the logic behind selecting this topology? It doesn't appear that you're using 9 links in the test code.